style.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+@import "/components/Header/header.css" ;
+@import "/components/alert/alert.css";
+@import "/components/Badge/badge.css";
+@import "/components/Buttons/button.css";
+@import "/components/Card/card.css";
+@import "/components/sideNav/sideNav.css";
+@import"/components/List/index.html";
+@import "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css";
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    font-family: 'Roboto', sans-serif;
+   
+    font-weight: 400;
+    font-size: large;
+}
+:root{
+    --primary-color:black;  
+    --primary-btn-color:rgb(10, 177, 10);
```

Comment:
You can use variables instead of directly writing color names in some places as you have used variables in some places which is a better practice.